pages/cloudflare/get-started.mdx

```diff
@@ -118,70 +143,42 @@ Add the following to the scripts field of your `package.json` file:
 - `npm run preview:worker`: Runs `build:worker` and then `dev:worker`, allowing you to quickly preview your app running locally in the Workers runtime, via a single command.
 - `npm run deploy`: Builds your app, and then deploys it to Cloudflare
 
-### 6. Add caching with Workers KV
+##### 7. Add caching with Workers KV
 
 See the [Caching docs](/cloudflare/caching) for information on enabling Next.js caching in your OpenNext project.
 
-### 7. Remove `@cloudflare/next-on-pages` (if necessary)
-
-If your Next.js app currently uses `@cloudflare/next-on-pages`, you'll want to remove it, and make a few changes.
+##### 8. Remove any `export const runtime = "edge";` if present
 
-#### Remove `export const runtime = "edge";`
+Before deploying your app, remove the `export const runtime = "edge";` line from any of your source files.
 
-Before deploying your app, remove the `export const runtime = "edge";` line from your `next.config.js` file.
 The edge runtime is not supported yet with `@opennextjs/cloudflare`.
 
-#### Add `.open-next` to `.gitignore`
+##### 9. Add `.open-next` to `.gitignore`
 
 You should add `.open-next` to your `.gitignore` file to prevent the build output from being committed to your repository.
 
-#### Uninstall `@cloudflare/next-on-pages`
-
-You should uninstall `@cloudflare/next-on-pages` and remove any references to it.
-
-In `package.json`:
+##### 10. Remove `@cloudflare/next-on-pages` (if necessary)
 
-```diff
-"scripts": {
--    "pages:build": "npx @cloudflare/next-on-pages",
```

Comment:
these are the scripts the C3 generates for users, but people can create their apps manually and not via C3 or update the name of such scripts, so I think that maybe it makes sense to just be generic here and say to remove any reference to the next-on-pages package and be done with it